src/Login.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import cn from 'classnames';
+import {
+  ChangeEvent,
+  FC,
+  useEffect,
+  useState,
+} from 'react';
+import { getUsers } from './api/users';
+import { User } from './types/User';
+
+export const Login: FC = () => {
+  const [email, setEmail] = useState('');
+
+  const [isLoading, setIsloading] = useState(false);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  const handleSubmit = (event: ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (email) {
+      setIsloading(true);
+    }
+  };
+
+  useEffect(() => {
+    if (isLoading) {
+      setTimeout(() => {
+        getUsers()
+          .then((users: any) => {
+            const { id } = users.find((user: User) => user.email === email);
+
+            localStorage.setItem('userId', id);
+          })
+          .then(() => window.location.reload());
+      }, 300);
+    }
+  }, [isLoading]);
+
+  return (
+    <section className="login mt-5">
+      <h1 className="login__title title is-3">Log in to open todos</h1>
+      <form onSubmit={handleSubmit}>
+        <div className="field">
+          <label
+            htmlFor="emailInput"
+            className="label"
+          >
+            Email
+          </label>
+          <p className="control has-icons-left">
```

Comment:
It's hard to read. Add spaces where needed
```suggestion
          </label>
          
          <p className="control has-icons-left">
```